benchmarking/src/lib.rs

```diff
@@ -600,7 +600,7 @@ macro_rules! benchmark_backend {
 // Every variant must implement [`BenchmarkingSetup`].
 //
 // ```nocompile
-// 
+//
```

Comment:
I think you are not using the same rustfmt version with our CI and causing this difference. Please manually revert this change